gdx/src/com/badlogic/gdx/graphics/g3d/particles/influencers/DynamicsModifier.java

```diff
@@ -415,11 +421,17 @@ public void update () {
 
 				float cosTheta = MathUtils.cosDeg(theta), sinTheta = MathUtils.sinDeg(theta), cosPhi = MathUtils.cosDeg(phi), sinPhi = MathUtils
 					.sinDeg(phi);
-				TMP_V3
-					.set(cosTheta * sinPhi, cosPhi, sinTheta * sinPhi)
-					.crs(positionChannel.data[positionOffset + ParticleChannels.XOffset],
-						positionChannel.data[positionOffset + ParticleChannels.YOffset],
-						positionChannel.data[positionOffset + ParticleChannels.ZOffset]).nor().scl(strength);
+				TMP_V3.set(cosTheta * sinPhi, cosPhi, sinTheta * sinPhi);
+				TMP_V1.set(positionChannel.data[positionOffset + ParticleChannels.XOffset],
```

Comment:
TMP_V1 now contains the (global) position of the particle to which we're applying the force. TMP_V3 is the vector representing the axis of rotation for.